packages/bot-web-ui/src/app/app.jsx

```diff
@@ -76,6 +77,7 @@ const App = ({ passthrough }) => {
         <Loading />
     ) : (
         <MobxContentProvider store={root_store_instance.current}>
+            <BlocklyLoding />
```

Comment:
will renaming this affect alot files? 

```suggestion
            <BlocklyLoading />
```